src/vehicle.cpp

```diff
@@ -3756,9 +3766,49 @@ void vehicle::idle(bool on_map) {
 
     if( on_map ) {
         update_time();
+        if(scoop_on){
+            operate_scoop();
+        }
+    }
+}
+void vehicle::operate_scoop()
+{
+    std::vector<int> scoops = all_parts_with_feature( "SCOOP" );
+    auto veh_points = get_points();
+    for( int scoop : scoops ) {
+        int chance_to_damage_item = parts[scoop].info().dmg_mod;
+        int max_pickup_size = parts[scoop].info().bonus;
+        const char *sound_msgs[] = {"Whirrrr", "Ker-chunk", "Swish", "Cugugugugug"};
+        sounds::sound( global_pos3() + parts[scoop].precalc[0], rng( 20, 35 ), sound_msgs[rng( 0, 3 )] );
+        for( const tripoint &position : veh_points ) {
```

Comment:
That area thing is a dealbreaker for me.
Having spikes and wing mirrors on the other side of the vehicle pick up stuff isn't acceptable.